packages/patternlab/styleguide/source/assets/js/modules/mapLeaflet.js

```diff
@@ -113,26 +124,54 @@ export default (function (window,document, $) {
       .addTo(mymap)
       .bindPopup(compiledTemplate(infoWindow));
 
-      $(mymarker._icon).on('click', function(e){
-        e.stopPropagation();
-        e.preventDefault();
-      });
+      $(mymarker._icon).on('keydown', function(e){
 
-      $(mymarker._icon).on('focus', function(e){
-        if (window.stopFocus) {
+        if (e.keyCode == 16 || e.keyCode == 9) {

```

Comment:
Please use .code and only use .keyCode as a fallback instead, as it's deprecated https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode